model/k8s/metrics.yaml

```
@@ -63,6 +63,28 @@ groups:
       - ref: network.interface.name
       - ref: network.io.direction
 
+  # k8s.container.* metrics
+  - id: metric.k8s.container.status.state
+    type: metric
+    metric_name: k8s.container.status.state
+    stability: experimental
+    brief: "The state of the container"
```

I think we should be aligned with https://github.com/open-telemetry/semantic-conventions/blob/main/model/k8s/metrics.yaml#L461:
```suggestion
    brief: "Describes the number of K8s containers that are currently in a given state"
```